release-notes/release-notes-2-10.md

```diff
@@ -141,6 +141,9 @@ For details on both see the commit history in GitHub.
 
 * It is recommended to use a chromium-based browser such as Google Chrome or Microsoft Edge, or Apple Safari for the Egeria React UI. Some parts of the UI experience such as Dino currently experience problems with Firefox. See [odpi/egeria-react-ui#96](https://github.com/odpi/egeria-react-ui/issues/96) .
 * When running the 'Understanding Platform Services' lab, ensure you run the 'egeria-service-config' notebook first and do not restart the python kernel before running this lab. See [#4842](https://github.com/odpi/egeria/issues/4842) .
+* The governance samples currently do not work. See See [#5300](https://github.com/odpi/egeria/issues/5300) and See [#5299](https://github.com/odpi/egeria/issues/5299) for more details
```

Comment:
This comment is not accurate - only the Governance Leadership Sample is not working.  The issue recorded on the create governance zone sample is a user error.

Reply:
The CLI sampledoes not work if run with the default userid as the code makes use of 'juleskeeper' as per https://github.com/odpi/egeria/issues/5300#issuecomment-853094286 . That is now updated in master only. 

Since it's minor it's probably ok to remove this bullet point and make the last bullet point more generic

Reply:
update added